### ISSUE TITLE
Geolocation error

### ISSUE THREAD

**Issue by garthtee** [electron/electron#13942]
* Electron Version: 2.0.6
* Operating System (Platform and Version): Windows 
* Last known working Electron version: Unknown

**Expected Behavior**
Using the `navigator.geolocation.getCurrentPosition` method, it should return a success and a position 
 object which contains a coords object. When I access it using a browser, it works as expected, using the same code but on chrome (Version 68.0.3440.84) instead of the electron application it works as expected. It does not work on my Electron application.

**Actual behavior**
The error function gets called and I receive the following error: { Code: 2, Message: "Network location provider at 'https://www.googleapis.com/' : Returned error code 403" }. Code 2 meaning POSITION_UNAVAILABLE.

**To Reproduce**
The error can be reproduced at this repo: https://github.com/garthtee/the-weather.git. Commit 94e3e290ce2d7ae5f6ab0c7e171fd972bf2b4394.

Setup my repo:
```sh
# Install dependencies
$ npm install

# Serve with hot reload at localhost:8080
$ npm run dev-server

# Start the electron applcation
$ npm run dev-app
```

**Comment by MarshallOfSound** [comment]
Duplicate of #9420

TLDR is this is expected behavior, for API's backed by google services you need to provide a google API key

**Comment by garthtee** [comment]
I have added the Google API as the docs suggest, this hasn't solved the issue

**Comment by MarshallOfSound** [comment]
Please see https://github.com/electron/electron/blob/master/docs/api/environment-variables.md#google_api_key

> By default, a newly generated Google API key may not be allowed to make geocoding requests. To enable geocoding requests, visit this page.

**Comment by garthtee** [comment]
Yes I have done this, I am able to access Google API features whilst running the application in a web browser, not on electron.

**Comment by ebizcoAU** [comment]
I also have done the same by going thru all the steps and got exactly the same problem.
Using 
Electron v5.0.2
Node v10.15.0 
I placed process.env.GOOLE_API_KEY = 'AIzaSys...'; at beginning of index.js
and confirmed with console.log(process.env.GOOLE_API_KEY); to be correctly passed in.
I was even blindly enable all 9 APIs for GGOLE_API_KEY to test it all.

Result still does not work.
I believe the electron reads the default built-in GOOGLE_API_KEY and ignore any setting in process.env.GOOGLE_API_KEY.

This problem has been around ON/OFF a long time and no one has a permanent fixed... Please help! if u got it working.

**Comment by promosjotte** [comment]
Hey Guys,

I have the same problem... process.env.GOOGLE_API_KEY in index.js is set, application running in Chrome is getting position. Application running in electron does not get location...

**Comment by tomasszabo** [comment]
I had the same problem but solved it through **ENABLING** Google's Geolocation API for my project on [Geolocation API Management Site](https://console.developers.google.com/apis/library/geolocation.googleapis.com) in Google's Developer Console. Before enabling, I had the api key correctly set in Electron but Google ignored requests from Electron. After enabling the Geolocation API in my project, it started to work.

**Comment by vincephan** [comment]
> I also have done the same by going thru all the steps and got exactly the same problem.
> Using
> Electron v5.0.2
> Node v10.15.0
> I placed process.env.GOOLE_API_KEY = 'AIzaSys...'; at beginning of index.js
> and confirmed with console.log(process.env.GOOLE_API_KEY); to be correctly passed in.
> I was even blindly enable all 9 APIs for GGOLE_API_KEY to test it all.
> 
> Result still does not work.
> I believe the electron reads the default built-in GOOGLE_API_KEY and ignore any setting in process.env.GOOGLE_API_KEY.
> 
> This problem has been around ON/OFF a long time and no one has a permanent fixed... Please help! if u got it working.

I got it working finally...  if you work with localhost,  set Application Restriction to None..

**Comment by chris1234p** [comment]
For me I had to add the env.GOOGLE_API_KEY, setup a paid google account with geolocation enabled, and update to electron 6 from electron 5.

**Comment by NingerJohn** [comment]
> > I also have done the same by going thru all the steps and got exactly the same problem.
> > Using
> > Electron v5.0.2
> > Node v10.15.0
> > I placed process.env.GOOLE_API_KEY = 'AIzaSys...'; at beginning of index.js
> > and confirmed with console.log(process.env.GOOLE_API_KEY); to be correctly passed in.
> > I was even blindly enable all 9 APIs for GGOLE_API_KEY to test it all.
> > Result still does not work.
> > I believe the electron reads the default built-in GOOGLE_API_KEY and ignore any setting in process.env.GOOGLE_API_KEY.
> > This problem has been around ON/OFF a long time and no one has a permanent fixed... Please help! if u got it working.
> 
> I got it working finally... if you work with localhost, set Application Restriction to None..

May I know what is the Application Restriction meaning?
